Documentation-Review-Process.md

```diff
@@ -0,0 +1,164 @@
+changes and add comments explaining how wording should be changed. It is then
+up to the PR creator to apply these changes to their original PR, rebasing as
+necessary to ensure a clean `git(1)` history. Once the creator has applied the
+changes they need to re-push their branch and add a comment to the PR asking
+for a re-review. This process can continue effectively indefinitely until the
+Documentation Team is happy with the final result.
+
+## Justification for different process
+
+The documentation review process is different for the code review process for
+the following reasons:
+
+### Users should always expect accurate Documentation
+
+Just like code changes, some documentation changes have to be merged as soon
+as possible to ensure users have the most current and accurate information.
+
+But documentation changes tend to be more urgent - consider a "Priority 1"
+code issue. This will need to be fixed with all urgency. But it would also
+need to be documented clearly (and quickly). If the P1 issue cannot be fixed
+quickly, the documentation needs to be changed fast to ensure users are aware
+of the issue and are provided with suitable workarounds.
+
+Turning the problem round, if documentation is *not* kept current, users will
+have incorrect information and may end up creating problems for themselves, or
+worse become disaffected with the project.
+
+In summary, documentation PRs should not sit in the PR backlog for any longer
+than necessary.
+
+### Some documentation PRs can be fast-tracked
+
+If a PR makes a minor change to one or more documents, it may not justify
+passing to the Documentation Team for formal review and approval.
+
+### Formal doc reviews can be very time-intensive
+
+Since a full document review can be a slow and pain-staking process, and since
+the number of professional technical writers on the Documentation Team is
+relatively small compared to the number of developers, it makes sense to only
+perform such reviews on the documents that justify the effort.
+
+## Decision Authority
+
+If a Documentation Team member believes a full review is required, their
+decision is final.
+
+In most other cases, a project maintainer will decide whether a full review is
+required or not.
+
+If project maintainers cannot decide whether the lightweight process should be
+followed, the full process should be used instead since the uncertainty
+implies "more eyes" should see the changes.
+
+## Determine if a full document review is required
+
+### Exempt
+
+If a PR contains documentation changes that **only** relate to one or more of
+the following categories, it does not need to be formally reviewed and
+approved by the Documentation Team:
+
+- Changes to code blocks
+
+   The PR only modifies code blocks embedded in the document.
+
+- "Typo" fixes
+
+  The PR only fixes spelling mistakes and whitespace issues.
+
+- Formatting
+
+  The PR only modifies the layout of the existing text (for example changing
```

Comment:
Lines 117-120 suggestions (missing a closing parenthesis): 

The PR only modifies the layout of the existing text. For example, you update words to render them in a ``fixed font`` or change a [Note](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes) to fit the standard formatting conventions.